wandb/sdk/internal/internal_api.py

```diff
@@ -2126,15 +2126,16 @@ def create_artifact(
         aliases=None,
         distributed_id=None,
         is_user_created=False,
+        enable_digest_deduplication=False,
     ):
-        # TODO: Ignore clientID and sequenceClientID if server can't handle it
         _, server_info = self.viewer_server_info()
         max_cli_version = server_info.get("cliVersionInfo", {}).get(
             "max_cli_version", None
         )
         can_handle_client_id = max_cli_version is None or parse_version(
             "0.11.0"
         ) <= parse_version(max_cli_version)
+        can_handle_dedupe = max_cli_version is None or parse_version("0.12.10")
```

Comment:
```suggestion
        can_handle_dedupe = max_cli_version is None or parse_version(
            "0.12.10"
        ) <= parse_version(max_cli_version)
```

Reply:
Woah, I didn't know you could suggest directly on GitHub. Sick. Thanks!